internal/testprotos/desc_test1.proto

```diff
@@ -88,6 +88,9 @@ message AnotherTestMessage {
 		// Comment for doors
 		optional string doors = 3;
 	}
+	// Comment for WithOptions
+	optional group WithOptions = 9 [deprecated = true] {
```

Comment:
Interesting. I thought groups just specified their options inside the curly brace. I didn't know you could independently apply _field_ options to them.